packages/loot-core/src/server/transactions/transaction-rules.ts

```
@@ -614,6 +615,29 @@ export function conditionsToAQL(
           }),
         };
 
+      case 'hasPeople':
+        const peopleValues = [];
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)